pkg/test/resourcefixture/testdata/basic/vertexai/v1beta1/vertexaiendpoint/vertexaiendpointencryptionkey/update.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: vertexai.cnrm.cloud.google.com/v1beta1
+kind: VertexAIEndpoint
+metadata:
+  name: vertexaiendpoint-${uniqueId}
+  labels:
+    label-one: "value-one"
+spec:
+  displayName: "vertexai-endpoint-encryptionkey-updated"
+  description: "example vertex AI endpoint with encryption key"
+  location: us-central1
+  region: us-central1
```

Comment:
Missed this before, but why do we have location & region?

Reply:
Now I remembered, I tried to ignore the field before. But it looks like it is required by the Terraform (although not directly used in the `resource_vertex_ai_endpoint.go`). Ignoring the field would lead to the following error.

Maybe we need to patch the Terraform code to use "region" instead of "location".

```
{"severity":"info","timestamp":"2024-04-29T18:17:30.668Z","msg":"resource is not ready","kind":"VertexAIEndpoint","name":"sdliqpjzcxxdcneitslqvrjxztykvlg","conditions":[{"lastTransitionTime":"2024-04-29T18:10:22Z",
"message":"Update call failed: error fetching live state: error reading underlying resource: summary: Cannot determine region: set in this resource, or set provider-level 'region' or 'zone'.","reason":"UpdateFailed
","status":"False","type":"Ready"}]}  
```